Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt

```diff
@@ -79,31 +89,64 @@ fun FilterChip(
 ) {
     val (selected, setSelected) = filter.enabled
     val backgroundColor by animateColorAsState(
-        if (selected) JetsnackTheme.colors.brand else JetsnackTheme.colors.uiBackground
+        if (selected) JetsnackTheme.colors.brandSecondary else JetsnackTheme.colors.uiBackground
     )
     val border = Modifier.fadeInDiagonalGradientBorder(
         showBorder = !selected,
         colors = JetsnackTheme.colors.interactiveSecondary,
         shape = shape
     )
     val textColor by animateColorAsState(
-        if (selected) JetsnackTheme.colors.textInteractive else JetsnackTheme.colors.textSecondary
+        if (selected) Color.Black else JetsnackTheme.colors.textSecondary
     )
+
     JetsnackSurface(
-        modifier = modifier
-            .height(28.dp)
-            .then(border),
+        modifier = modifier.height(28.dp),
         color = backgroundColor,
         contentColor = textColor,
         shape = shape,
         elevation = 2.dp
     ) {
+        val interactionSource = remember { MutableInteractionSource() }
```

Comment:
I thikn this could use [collectIsPressedAsState](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/interaction/PressInteraction.kt;l=81)

Reply:
You are right, I just made the adjustments